app/assets/stylesheets/homepage.scss

```diff
@@ -0,0 +1,40 @@
+$color: #EC4339;
+
+.title-box {
+   background-color: $color;
+   font-family: 'Alegreya Sans SC', sans-serif;
+   color: #ffffff;
+   padding: 10px;
+   font-weight: bold;
+   letter-spacing: 1.25px;
+   font-size: 60px;
+   font-size: 5vw;
```

Comment:
Property `font-size` already defined on line 10<br>Line should be indented 2 spaces, but was indented 3 spaces